### PR TITLE
Pro-actively detect whether user acquires tokens and move to the next…

### PR DESCRIPTION
**Pull request by agatsoh**
Solves #273

### CODE REVIEW

File: raiden_installer/web.py [hunk 1]
```diff
@@ -370,42 +371,10 @@ def _run_swap(self, **kw):
                 if total_service_token_balance < required.service_token:
                     raise ExchangeError("Exchange was not successful")
 
-                elif service_token_balance.as_wei > 0:
+                elif token_ticker == service_token.ticker and service_token_balance.as_wei > 0:
```
Comment (konradkonrad) [inline raiden_installer/web.py:374]: why does this not check for 
```suggestion
                elif token_ticker == service_token.ticker and service_token_balance.as_wei > required.service_token:
```
?

Comment (agatsoh) [reply raiden_installer/web.py:374]: One small change  `elif token_ticker == service_token.ticker and service_token_balance > required.service_token:`.(no as_wei) I will make the change.

**Comment by agatsoh** [comment]
I need get the tests in now

**Comment by manuelwedler** [comment]
closing because I created a new PR which contains these changes rebased on the latest develop #301

**Pull request closed without merging**